mltrace/cli/cli.py

```diff
@@ -307,3 +338,67 @@ def review(limit: int = 5, address: str = ""):
     # Print component runs
     for component, count in component_counts[:limit]:
         show_info_card(component.id, count, len(outputs))
+
+
+@mltrace.command("components")
+@click.option("--owner", help="Owner of components")
+@click.option("--tag", help="Tag of components")
+@click.option("--address", help="Database server address")
+def components(owner: str = "", tag: str = "", address: str = ""):
+    """
+    Command to list the components with options to filter by tag or owner.
+    """
+    if address and len(address) > 0:
+        set_address(address)
+
+    # Make return result
+    components = []
+    if owner:
+        components = get_components_with_owner(owner)
+    elif tag:
+        components = get_components_with_tag(tag)
+    else:
```

Comment:
What is this else case for? I'm not sure it's necessary

Reply:
It is for the general case of if they did not set an owner or a tag, list all components.